CONTRIBUTING.md

```diff
@@ -124,6 +124,7 @@ Any changes to the [roles](roles/) tree or to the [playbook](playbook.yml) file
 The benchmark-operator container can be built using [podman](https://podman.io/) and pushed to a public repository.
 The public repository could be [quay](https://quay.io) in which case you'll need to:
 
+#### v0.1
```

Comment:
Hey @jeniferh, the docs for the v0.1 tag are correct, I think we shouldn't include the v0.1 section in the master branch,

I'd just move the comments from the "v0.1" you've created here to the "The operator container image" section

Reply:
Good point, I will adjust so the master version only documents the current method since the v0.1 tag covers the old method.